content/release/ReleaseNotes370.md

```diff
@@ -22,6 +22,8 @@ PDF Compliance Reports now include the latest Payment Card Industry (PCI) Securi
 
 Sensitive data masking is now available for all language agents.
 
+Library policy features have been extended to support open source licensing policy. Library dependency tree powered by a new command line interface has been introduced.
+
```

Comment:
Thats Kristen, this covers the OSS features

Reply:
Thumbs up from me (Damian)